### DISCUSSION TITLE
What's going on with the Release of v8.3.4?

### DISCUSSION THREAD

**Discussion by herbetom** [grafana/grafana#44219] [General]
It seems like [v8.3.4](https://github.com/grafana/grafana/releases/tag/v8.3.4) keeps getting released over and over?

![image](https://user-images.githubusercontent.com/15121114/150146931-98b59689-4d3b-40a3-a1b3-dbde4d3265ed.png)

**Comment by herbetom** [comment]
If this is about GitHub showing [v7.5.13](https://github.com/grafana/grafana/releases/tag/v7.5.13) as the latest release, i think that's because the tag the release is created from ist younger.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
